user-guide/Advanced_Functionality/Databases/Specifying_TTL_overrides.md

```diff
@@ -9,7 +9,9 @@ The "time to live" (TTL) determines how long specific data is retained before be
 This page explains how to configure TTL settings to optimize your database performance and resource utilization.
 
 > [!NOTE]
-> If a DMS uses [Storage as a Service](xref:STaaS), the default TTL is automatically applied for most database records. The configuration in Cube is only taken into account for real-time trending.
+>
+> - If a DMS uses [Storage as a Service](xref:STaaS), the default TTL is automatically applied for most database records. The configuration in Cube is only taken into account for real-time trending.
+> - TTL settings are automatically configured as part of [DaaS](xref:Creating_a_DMS_in_the_cloud). From DataMiner 10.3.0 [CU15]/10.4.0 [CU3]/10.4.6 onwards<!--RN 39173-->, these settings are no longer available for manual adjustment on the *System settings* > *time to live* tab in System Center when you are using a DaaS system.
```

Comment:
Just to avoid any confusion, another small change:
```suggestion
> - TTL settings are automatically configured if you use [DaaS](xref:Creating_a_DMS_in_the_cloud). From DataMiner 10.3.0 [CU15]/10.4.0 [CU3]/10.4.6 onwards<!--RN 39173-->, these settings are no longer available for manual adjustment on the *System settings* > *time to live* tab in System Center when you are using a DaaS system.
```